Config/DefaultEngineBase.ini

```diff
@@ -21,6 +21,9 @@ bUseFixedFrameRate=True
 FixedFrameRate=100.000000
 GameSingletonClassName=/Script/RapyutaSimulationPlugins.RRGameSingleton
 
+[/Script/NavigationSystem.RecastNavMesh]
+RuntimeGeneration=Dynamic
```

Comment:
how this affect?

Reply:
This is for the future use of AI nav if required